lesson5/src/com/tms/Task11.java

```diff
@@ -1,14 +1,19 @@
 package com.tms;
 
-//Имеется целое число, определить является ли это число простым, т.е делиться без остатка только на 1 и себя.
+//*Имеется целое число, определить является ли это число простым, т.е делиться без остатка только на 1 и себя.
 
 public class Task11 {
     public static void main(String[] args) {
-        int x = (int) (Math.random() * 4);
-        if (x % x == 0 & x % 1 == 0) {
-            System.out.println(x + " число является простым");
-        } else {
-            System.out.println(x + " число является составным");
+        int n = (int) (Math.random() * 20)+1;
+        int i;
+        for (i = 2; i < n; i++) {
+            n = (int) (Math.random() * 20)+1;
+            if (n % i == 0 ) {
+                System.out.println("Число " + n + " - является составным");
+            } else {
+                System.out.println("Число " + n + " - является простым");
+            return;
```

Comment:
move return to if(){...} statement. You need to stop your cycle in case of not simple number but not in other case